File: Ejercicio 9_Primer_Departamental.py
class Operaciones:

    def __init__(self):
        self.numeros = []

    def ingresar_numeros(self):
        self.numeros = list(map(float, input("Ingresa 5 numeros separados por espacio: ").split()))

    def calcular(self):
        suma = sum(self.numeros)
        media = suma / len(self.numeros)
        maximo = max(self.numeros)
        minimo = min(self.numeros)

        print("Numeros ingresados:", self.numeros)
        print("Suma:", suma)
        print("Media:", media)
        print("Maximo:", maximo)
        print("Minimo:", minimo)


op = Operaciones()
op.ingresar_numeros()
op.calcular()

File: Ejercicio 9_Primer_Departamental.cpp
#include <iostream>
using namespace std;

int main() {
    
    float numeros[5];
    float suma = 0;
    float promedio;
    float maximo, minimo;

    for(int i = 0; i < 5; i++) {
        cout << "Ingrese el numero " << i + 1 << ": ";
        cin >> numeros[i];
        suma = suma + numeros[i];
    }

    promedio = suma / 5;

    maximo = numeros[0];
    minimo = numeros[0];


    for(int i = 1; i < 5; i++) {
        if(numeros[i] > maximo) {
            maximo = numeros[i];
        }
        if(numeros[i] < minimo) {
            minimo = numeros[i];
        }
    }

    cout << "\nResultados:" << endl;
    cout << "Suma: " << suma << endl;
    cout << "Promedio: " << promedio << endl;
    cout << "Maximo: " << maximo << endl;
    cout << "Minimo: " << minimo << endl;

    return 0;
}